ipi/inputs/motion/instanton.py

```diff
@@ -49,18 +49,6 @@ class InputInst(InputDictionary):
         )
     }
 
```

Comment:
Why this inputAtributre has been eliminated?

Reply:
the whole block is completely repeated in the master branch, take a look :)